tensorflow/core/kernels/matmul_op_test.cc

```diff
@@ -307,13 +314,13 @@ TYPED_TEST_P(FusedMatMulWithBiasOpTest, MatMul1x256x1) {
 
 TYPED_TEST_P(FusedMatMulWithBiasOpTest, MatMul256x256x256WithActivation) {
   for (const string& activation : {"Relu", "Relu6", "Elu", "LeakyRelu"}) {
-    this->VerifyConv2DWithBiasAndActivation(256, 256, 256, false, false,
+    this->VerifyConv2DWithBiasAndActivation(256, 128, 64, false, false,
```

Comment:
Same here, why the change of dimension sizes? And if necessary, then also update the test name.